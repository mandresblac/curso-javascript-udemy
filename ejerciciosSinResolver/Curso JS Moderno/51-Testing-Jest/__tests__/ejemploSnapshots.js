const cliente = {
    nombre: "Manuel 2",
    balance: 500,
    tipo: "Premium"
};

describe('Testing al cliente', () => {

    // Prueba 1
    test('Es Manuel Blanco', () => {
        expect(cliente).toMatchSnapshot();
    });

});