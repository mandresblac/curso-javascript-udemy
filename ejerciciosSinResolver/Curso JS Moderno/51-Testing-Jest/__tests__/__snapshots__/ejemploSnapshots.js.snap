// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al cliente Es Manuel Blanco 1`] = `
{
  "balance": 500,
  "nombre": "Manuel 2",
  "tipo": "Premium",
}
`;
